css/_videolayout_default.scss

```diff
@@ -65,6 +65,15 @@
             float: left;
             pointer-events: all;
         }
+
+        /**
+         * Need to overwrite the background for the top toolbar dark theme div
+         * wrapper needed before we're able to move all top toolbar indicators
+         * creation to react.
+         */
+        .ckAJgx {
+            background: none;
+        }
```

Comment:
You'll have to apply styling in addition to ".ckAJgx a" for the volume slider because right now it's too dark to see.